src/wp-admin/includes/class-wp-media-list-table.php

```diff
@@ -425,8 +425,18 @@ public function column_cb( $item ) {
 	public function column_title( $post ) {
 		list( $mime ) = explode( '/', $post->post_mime_type );
 
+		$attachment_id = $post->ID;
+
+		if ( 'video' === $mime || 'audio' === $mime ) {
+			$thumbnail_id = get_post_thumbnail_id( $post->ID );
```

Comment:
Should use already defined variable. 


```suggestion
			$thumbnail_id = get_post_thumbnail_id( $attachment_id );
```